attest/attest-tool/attest-tool.go

```diff
@@ -25,6 +25,7 @@ var (
 	nonceHex    = flag.String("nonce", "", "Hex string to use as nonce when quoting")
 	randomNonce = flag.Bool("random-nonce", false, "Generate a random nonce instead of using one provided")
 	useSHA256   = flag.Bool("sha256", false, "Use SHA256 for quote operatons")
+	ekIndex     = flag.Int("ek-index", 0, "Which EK to use for credential activation. The first EK is at index 0")
```

Comment:
nit: we don't guarantee ordering so drop this flag and just pick the first one?

Reply:
Done.